example-mojaloop-backend/values.yaml

```diff
@@ -326,48 +197,25 @@ centralledger-obj:
   ## Enable persistence using Persistent Volume Claims
   ## ref: https://kubernetes.io/docs/user-guide/persistent-volumes/
   ##
-  persistence:
-    ## @param persistence.enabled Enable MongoDB(&reg;) data persistence using PVC
-    ##
-    enabled: false
+  # persistence:
+  #   ## @param persistence.enabled Enable MongoDB(&reg;) data persistence using PVC
+  #   ##
+  #   enabled: false
 
 ## TODO: Add 
 cep-mongodb:
```

Comment:
Same comment applies here for possibly merging both Mongo DBs.

Reply:
@mdebarros yeah same reply , lets do this in a separate issue after merging this PR

Reply:
Created a story for this: 

Separate story for MongoDB consolidation --> https://github.com/mojaloop/project/issues/3104